BoardConfigPlatform.mk

```diff
@@ -70,6 +70,9 @@ BOARD_CACHEIMAGE_FILE_SYSTEM_TYPE := ext4
 BOARD_ROOT_EXTRA_FOLDERS := ocm
 TARGET_USERIMAGES_USE_EXT4 := true
 
+### POWER
+TARGET_USE_CUSTOM_POWERHINT ?= FALSE
```

Comment:
The `powerhint.json` is currently not included in a poplar build (lilac is probably fine, but haven't checked). Guessing this should be changed from `FALSE` to `false` to match the conditional. Will verify later today.

Reply:
@linckandrea I guess this should have really been `false`?

Reply:
fixed